### DISCUSSION TITLE
不止顺序执行

### DISCUSSION THREAD

**Discussion by raccoon666666** [shuax/MouseInc.Settings#376] [Ideas]
有些重复执行的动作只能`多次触发`或者`action写很长(但也是有限次数)`, 能不能出一个类似`['goto',2]`的`action`, 这样直接跳转到第2个`action`, 从而实现无限循环, 脑洞大点还能实现像`['set','a','1'], ['if','a','lt','0'],['endif']`的`action`, 用类似汇编指令的方法实现程序的顺序
、分支、循环三大执行顺序，到时候MouseInc能做什么我已经不敢想了

**Comment by shuax** [comment]
MouseInc3有打算支持lua脚本。

**Reply by raccoon666666** [reply]
芜！期待
